src/components/aktivitetskrav/vurdering/OppfyltAktivitetskravSkjema.tsx

```diff
@@ -41,55 +26,26 @@ export const OppfyltAktivitetskravSkjema = ({
   setModalOpen,
   aktivitetskravUuid,
 }: OppfyltAktivitetskravSkjemaProps) => {
-  const vurderAktivitetskrav = useVurderAktivitetskrav(aktivitetskravUuid);
-
-  const submit = (values: OppfyltAktivitetskravSkjemaValues) => {
-    const createAktivitetskravVurderingDTO: CreateAktivitetskravVurderingDTO = {
-      status: AktivitetskravStatus.OPPFYLT,
-      beskrivelse: values.beskrivelse,
-      arsaker: [values.arsak],
-    };
-    vurderAktivitetskrav.mutate(createAktivitetskravVurderingDTO, {
-      onSuccess: () => setModalOpen(false),
-    });
-  };
+  const { createDto, validateArsak, validateBeskrivelse } =
+    useAktivitetskravVurderingSkjema(AktivitetskravStatus.OPPFYLT);
 
-  const validate = (values: Partial<OppfyltAktivitetskravSkjemaValues>) => {
-    return {
-      [vurderAktivitetskravArsakFieldName]: !values.arsak
-        ? texts.missingArsak
-        : undefined,
-      [vurderAktivitetskravBeskrivelseFieldName]: validerTekst({
-        value: values.beskrivelse || "",
-        maxLength: vurderAktivitetskravBeskrivelseMaxLength,
-      }),
-    };
-  };
+  const validate = (values: Partial<OppfyltAktivitetskravSkjemaValues>) => ({
```

Comment:
Kult, måtte lese meg opp på Partial 🙌🏼  https://dev.to/smpnjn/how-the-typescript-partial-type-works-2klj